docs/reference/modules/ui.md

```diff
@@ -0,0 +1,221 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/ui`
+
+ The `@apostrophecms/ui` module provides the components and mixins for creating the admin UI. It also provides the UI styling with some built-in theming capabilities that can be leveraged to create a custom admin experience.
```

Comment:
I would add: "this module has no impact on the experience of ordinary logged-out site visitors."

Reply:
Added.